src/components/themeModeIcon/themeModeIcon.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from './themeModeIcon.module.scss';
+import { useRef } from 'react';
+import { getIsDarkMode, setDarkMode, removeDarkMode } from '../../themeMode/ThemeMode';
+import IconButton from '../IconButton/IconButton';
+import clsx from 'clsx';
+import { FiSun, FiMoon } from 'react-icons/fi';
+
+const ThemeModeIcon = () => {
+  const lightModeIconRef = useRef<HTMLButtonElement>(null);
+  const darkModeIconRef = useRef<HTMLButtonElement>(null);
+  const isDarkModeFromLocalStorage = getIsDarkMode();
+  isDarkModeFromLocalStorage && document.body.classList.add('theme-dark');
+
+  const handleOnclick = () => {
```

Comment:
handleOn what click ?
Names should suggest what this should be doing
hadndleChangeTheme or handleOnChangeThemeButtonClick